redfish/rules/dell_redfish_1.17.0.yml

```diff
@@ -0,0 +1,431 @@
+Traverse:
+  Root: /redfish/v1
+  Excludes:
+  - /JsonSchemas
+  - /Accounts
+  - /Certificates
+  - /Jobs
+  - /Logs
+  - /Registries
+  - /Roles
+  - /Sessions
+  - /Settings
+  - /AccountService
+  - /CertificateService
+  - /EventService
+  - /JobService
+  - /LicenseService
+  - /LogServices
+  - /SessionService
+  - /TaskService
+  - /TelemetryService
+  - /UpdateService
+  - /Power/
+  - /Power#/
+  - /Thermal#/
+  - /Assembly#/
+  - /redfish/v1/Chassis/$
+Metrics:
+- Path: /redfish/v1/Chassis/System.Embedded.1/NetworkAdapters/NIC.Embedded.1

```

Comment:
Use placeholders as follows:
`/redfish/v1/Chassis/{chassis}/NetworkAdapters/{nic}`

Placeholders are used as labels for metrics.

Reply:
I have reworked the rules for redfish v1.17 based on your comments.